synapse/lib/storm.lark

```diff
@@ -143,10 +143,13 @@ funcargs: "(" _WS? (VARTOKN (_WS? "," _WS? VARTOKN)* _WS? )? ")"
 stormfunc: "function" _WS VARTOKN _WS? funcargs _WS? "{" query "}"
 
 stormcmd: CMDNAME [_WS stormcmdargs]
-stormcmdargs: _WS? _cmdargv (_WS _cmdargv)*
-_cmdargv: baresubquery | DOUBLEQUOTEDSTRING | SINGLEQUOTEDSTRING | NONCMDQUOTE
-// An unquoted string that's a command argument
-NONCMDQUOTE: /(?!['\"])[^\s|}]+/
+stormcmdargs: _WS? _cmdargv (["="] _WS? _cmdargv)*
```

Comment:
It should probably be ("=" | _WS) (no question mark). We don't want to allow an equals and a space.